source/assets/scripts/recipe-add.js

```diff
@@ -43,6 +43,21 @@ function addNewRecipe() {
 
     const servingSizeField = document.getElementById('servingSize').value;
     recipe.servings = servingSizeField;
+
+
+    // Ensure that there are no extreme values as the input:
+    if (recipe.readyInMinutes.length > 3 || recipe.servings.length > 3) {
+
+      // Get the InvalidInput DIV
+      var x = document.getElementById("InvalidInput");
+
+      // Add the "show" class to DIV
+      x.className = "show";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/12DaysOfCoding/WeTheFoodies/pullRequest?prid=9413726)
```suggestion
      x.className = 'show';
```